<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataTypeWidget</class>
 <widget class="QWidget" name="DataTypeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblType">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbType"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblData">
       <property name="text">
        <string>Data</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="layoutData">
       <item>
        <widget class="QLineEdit" name="editConstant"/>
       </item>
       <item>
        <widget class="QLineEdit" name="editFunction">
         <property name="enabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbVariable"/>
       </item>
       <item>
        <widget class="QFrame" name="frameTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QPushButton" name="btnTableAdd">
              <property name="text">
               <string>Add row</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnTableDel">
              <property name="text">
               <string>Delete row</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="tblTable"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="framePiecewise">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QPushButton" name="btnPiecewiseAdd">
              <property name="text">
               <string>Add row</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnPiecewiseDel">
              <property name="text">
               <string>Delete row</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QTableView" name="tblPiecewise"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
